lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -108,8 +108,7 @@ private function includeFileForId($id)
     {
         $fileName = $this->getFilename($id);
 
-        // note: error suppression is still faster than `file_exists`, `is_file` and `is_readable`
-        $value = @include $fileName;
+        $value = file_exists($fileName) ? @include $fileName : null;
```

Comment:
If error suppression really is faster than file_exists (which would make sense) maybe better way would be:

```php
set_error_handler(function ($errno, $errstr, $errfile, $errline) { /* do nothing */ });
$value = include $fileName;
restore_error_handler();
```
? It will also ignore errors from included file, but that shouldn't make any difference.